json-rpc/src/tests/unit_tests.rs

```diff
@@ -378,7 +378,8 @@ fn test_get_transactions() {
             let version = base_version + i as u64;
             assert_eq!(view.version, version);
             let (tx, status) = &mock_db.all_txns[version as usize];
-            assert_eq!(view.hash, tx.hash().to_string());
+            assert_eq!(view.hash.len(), 64);
```

Comment:
I think this is unnecessary now, due to the second part.  

we should really get rid of to_string() api on that hash() , at least rename it to to_short_string() or something. otherwise there might be a lot of places were misused.

Reply:
deleted hash len assertion